pyfronius/__init__.py

```diff
@@ -304,6 +311,8 @@ async def fetch(
             requests.append(self.current_storage_data(i))
         for i in device_inverter:
             requests.append(self.current_inverter_data(i))
+        for i in device_inverter:

```

Comment:
Are the 3P inverters unique or coinciding with the normal inverters? Or are they exclusively either 3P or non-3P? I am asking basically whether users should be able to configure 3P inverters and normal inverters separately or if it makes sense to always query both endpoints for the given list of inverters.


Concretely: can it be that the user has only a 1P inverter and this addition now always adds failing requests? If you have a 3P inverter, do you need the requests to the 1P inverter above?

Reply:
I don't have a 1P inverter to test, but I did some internet research before implementing. 

The request should not fail for 1P inverters, but return an empty data object.

See samples here:
https://github.com/victronenergy/dbus-fronius/blob/master/documents/solar_api_samples.txt

Reply:
I see! In that case this configuration seems sensible to me.